src/libPMacc/include/mpi/MPIReduce.hpp

```diff
@@ -52,6 +52,7 @@ struct MPIReduce
     {
         if (isMPICommInitialized)
         {
+            /* exception inside the destructor are forbidden */
```

Comment:
My point did not get clear: I also meant that the comment does not have to be changed. It was only a clarification.